File: leetcode/java/9.Palindrome.java
class p{
    static boolean isPalindrome(int x){
        int n=x;
        if(x<0){
            return false;
        }
        int rev=0;
        while(n>0){
            rev= rev*10 + n%10;
            n/=10;
        }
        if(x==rev){
            return true;
        }else{
            return false;
        }
    }

    public static void main(String[] args) {
        System.out.println(isPalindrome(121));
    }
}

File: leetcode/kotlin/9.Palindrome.kt
class Solution{
    companion object{
        fun isPalindrome(x:Int):Boolean{
            var n:Int=x
            if(x<0)
                return false
            var rev:Int=0
            while(n>0){
                rev=rev*10+(n%10)
                n/=10
            }
            return x==rev
        }
    }
}

fun main(){
    print(Solution.isPalindrome(121))
}